pool/pool_manager.gno

```diff
@@ -30,6 +51,87 @@ func init() {
 	feeAmountTickSpacing[10000] = 200 // 1%
```

Comment:
It seems like tickSpacingMap has getter, why don't you also implement setter and use it instead of setting value directly `feeAmountTickSpacing[10000] = 200 // 1%`

Reply:
I thought the current implementation was clearer, because I couldn't make a proper name to this constant because I don't know exactly what value it refers to at the moment, and I felt that using a hardcoded value in the `Set` function would only add to the confusion. So, I let it them.